src/edu/csus/ecs/pc2/ui/AutoJudgeStatusFrame.java

```diff
@@ -222,4 +231,59 @@ public String getPluginTitle() {
         return "Auto Judge Status Frame";
     }
 
+    @Override
+    public void resetFrame() {
+        if(bigAutoJudgeStatusLabel != null) {
+            SwingUtilities.invokeLater(new Runnable() {
+                public void run() {
+                    bigAutoJudgeStatusLabel.setForeground(Color.black);
+                }
+            });
+           
+        }
+    }
+
+    @Override
+    public void setTimerFrameVisible(boolean bHow)
+    {
+        // The AJ Status frame is not controlled by the execution timer, but the AJ itself
```

Comment:
Seems like an incomplete comment, complete it.

Reply:
Cleared up the confusion, hopefully.